src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -95,15 +98,19 @@ private void bindIntakeControls() {
 	}
 
 	private void bindLauncherControls() {
-		launcherPodiumPresetButton.onTrue(
-				new SetAngleLaunchCommand(
-						s.launcherSubsystem,
-						LauncherSubsystem.SPEAKER_SHOOT_SPEED_RPM,
-						LauncherSubsystem.PODIUM_AIM_ANGLE));
-		launcherSubwooferPresetButton.onTrue(
-				new SetAngleLaunchCommand(
-						s.launcherSubsystem,
-						LauncherSubsystem.SPEAKER_SHOOT_SPEED_RPM,
-						LauncherSubsystem.SUBWOOFER_AIM_ANGLE));
+		// launcherPodiumPresetButton.onTrue(
+		//		new SetAngleLaunchCommand(
+		//				s.launcherSubsystem,
+		//				LauncherSubsystem.SPEAKER_SHOOT_SPEED_RPM,
+		//				LauncherSubsystem.PODIUM_AIM_ANGLE));
+		// launcherSubwooferPresetButton.onTrue(
+		//		new SetAngleLaunchCommand(
+		//				s.launcherSubsystem,
+		//				LauncherSubsystem.SPEAKER_SHOOT_SPEED_RPM,
+		//				LauncherSubsystem.SUBWOOFER_AIM_ANGLE));
```

Comment:
Curious, why were these deleted? (now we have no way to use presets).

Reply:
presets, deleted for now as intake is using the keybinds we needed, it will be re-instated if we need presets right before comp or if the keybinding thing is resolved.